publisher/start.sh

```diff
@@ -28,33 +28,44 @@ EORESTORE
     docker push ${DESTINATION_DOCKER_IMAGE}-roles-only
 
     docker start publishing_db_container
-    docker exec publishing_db_container cp /files/${backup_file} /dumps/db.backup
-
-    cat <<-EORESTORE | (docker exec -i publishing_db_container sh -c "cat >> restore.sh")
-      pg_restore /dumps/db.backup -U postgres -d ${DB_NAME}
-EORESTORE
 
     # we need to sanitize sensitive data first
-    docker run --name sanitizing_db_container -d --rm -v ${DUMPS_DIR}:/files \
-    -e POSTGRES_DB=${DB_NAME} -e POSTGRES_PASSWORD= postgres:10.5
-    # wait for "database system is ready to accept connections", hoping 4 sec is enough
-    sleep 4;
+    docker run --name sanitizing_db_container -d --rm \
+    -v ${DUMPS_DIR}:/files -v ${SQL_DIR}:/sql \
+    -e POSTGRES_DB=${DB_NAME} -e POSTGRES_PASSWORD= ${POSTGRES_IMAGE}
+    db_initialized=0
+    # waiting 5 mins for sanitizing_db_container to be initialized
+    for i in `seq 1 300`;
+    do
+      docker exec sanitizing_db_container psql -U postgres -c "\t" &> /dev/null && db_initialized=1 && break
+      echo -n .
+      sleep 1
+    done
+
+    echo ""
+
+    if [ "$db_initialized" -eq 0 ]
+    then
+        echo "Sanitizing DB was not initialized after 5 mins, stopping..."
```

Comment:
maybe change to 1 min? usually if it's not enough, than something goes really wrong